keystore/keyloader/hashicorp/vault_cli.go

```diff
@@ -36,11 +41,12 @@ func RegisterCLIParametersWithFlagSet(flags *flag.FlagSet, prefix, description s
 	if flags.Lookup(prefix+vaultConnectionStringFlag) == nil {
 		flags.String(prefix+vaultConnectionStringFlag, "", "Connection string (http://x.x.x.x:yyyy) for loading ACRA_MASTER_KEY from HashiCorp Vault"+description)
 		flags.String(prefix+"vault_secrets_path", defaultVaultSecretsPath, "KV Secret Path (secret/) for reading ACRA_MASTER_KEY from HashiCorp Vault"+description)
-		flags.String(prefix+"vault_tls_ca_path", "", "Path to CA certificate for HashiCorp Vault certificate validation"+description)
```

Comment:
as mentioned above, we should leave it because it is breaking change. and deal with it too.